src/styles/search.css

```diff
@@ -0,0 +1,33 @@
+.search-input {
+    width: 100%;
+    height: 50px;
+    border-radius: 100px;
+    font-size: 20px;
+    padding: 0 10px;
+    background-color: #BF0016;
+    border-color: white;
+    color: white;
+    font-weight: bold;
+    font-style: italic;
+}
+
+.search-button {
+    border-radius: 100px;
+    background-color: #BF0016;
+    font-weight: bold;
+    font-size: 20px;
+    border-color: white;
+    color: white;
+
+}
```

Comment:
Theres a lot of repetition here. You could combine shared properties into a declaration set for both of these classes:
```
.search-input,
.search-button {
  border-radius: 100px;
  font-size: 20px;
  background-color: #BF0016;
  border-color: white;
  color: white;
  font-weight: bold;
}

.search-input {
    width: 100%;
    height: 50px;
    padding: 0 10px;
    font-style: italic;
}
```

Then you could delete the `.search-button` class as all its properties are already covered